src/App.tsx

```diff
@@ -362,74 +363,76 @@ const App = () => {
+                        <span>
+                          <SideNavView
+                            presenceData={state.presenceData}
+                            currentRoomId={state.roomId}
+                            username={state.userMap[state.userId].username}
+                            spaceIsClosed={state.isClosed}
+                          />
+                          {/* Once we moved the sidebar to be position:fixed, we still
                       needed something to take up its space in the CSS grid.
                       This should be fixable via CSS, but sigh, it's 3 days before the event */}
-                        <div id="side-nav-placeholder" />
-                      </span>
-                    ) : (
-                      <button id="show-menu" onClick={showMenu}>
-                        <span role="img" aria-label="menu">
-                          🍔
+                          <div id="side-nav-placeholder" />
                         </span>
-                      </button>
-                    )}
-                    {modalView}
-                    <div id="main" role="main">
-                      {state.roomData[state.roomId] ? (
-                        <RoomView
-                          room={state.roomData[state.roomId]}
-                          userId={state.userId}
-                          roomData={state.roomData}
-                          presenceData={state.presenceData}
-                          inMediaChat={state.inMediaChat}
-                          keepCameraWhenMoving={state.keepCameraWhenMoving}
-                          textOnlyMode={state.textOnlyMode}
-                          mediaChatView={videoChatView}
-                          hasDismissedAModal={state.hasDismissedAModal}
+                      ) : (
+                        <button id="show-menu" onClick={showMenu}>
+                          <span role="img" aria-label="menu">
+                          🍔
+                          </span>
+                        </button>
+                      )}
+                      {modalView}
+                      <div id="main" role="main">
+                        {state.roomData[state.roomId] ? (
+                          <RoomView
+                            room={state.roomData[state.roomId]}
+                            userId={state.userId}
+                            roomData={state.roomData}
+                            presenceData={state.presenceData}
+                            inMediaChat={state.inMediaChat}
+                            keepCameraWhenMoving={state.keepCameraWhenMoving}
+                            textOnlyMode={state.textOnlyMode}
+                            mediaChatView={videoChatView}
+                            hasDismissedAModal={state.hasDismissedAModal}
+                          />
+                        ) : null}
+                        <ChatView
+                          messages={Object.values(state.messages.entities)}
```

Comment:
This converts the new `messages.entities` object back into the expected array of message objects for the chat view.